packer/install-dependencies.sh

```diff
@@ -376,7 +376,7 @@ for KUBERNETES_VERSION in ${K8S_VERSIONS}; do
     if [[ $KUBERNETES_VERSION == *"azs"* ]]; then
       HYPERKUBE_URL="msazurestackdocker/hyperkube-amd64:v${KUBERNETES_VERSION}"
     else
-      HYPERKUBE_URL="k8s.gcr.io/hyperkube-amd64:v${KUBERNETES_VERSION}"
+      HYPERKUBE_URL="upstream.azurecr.io/k8s/core/hyperkube-amd64:v${KUBERNETES_VERSION}"
```

Comment:
Just want to double check if we have all the versions we support today in MCR.

Reply:
see the comment https://github.com/Azure/aks-engine/pull/797#discussion_r286144437, not all the versions are available. We may need to block this until all the AKS supported versions are pushed to MCR.